.github/workflows/basic.yaml

```diff
@@ -25,9 +25,11 @@ jobs:
 
       - name: Install protoc
         uses: arduino/setup-protoc@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As per the action [doc](https://github.com/arduino/setup-protoc?tab=readme-ov-file#usage)
> The action queries the GitHub API to fetch releases data, to avoid rate limiting, pass the default token with the repo-token variable:
```
- name: Install Protoc
  uses: arduino/setup-protoc@v3
  with:
    repo-token: ${{ secrets.GITHUB_TOKEN }}
```